node/node_test.go

```diff
@@ -2401,7 +2406,7 @@ func TestNode_GetAccountAccountDoesNotExistsShouldRetEmpty(t *testing.T) {
 
 	n, _ := node.NewNode(
 		node.WithAccountsAdapter(accDB),
-		node.WithPubkeyConverter(createMockPubkeyConverter()),
+		node.WithAddressPubkeyConverter(createMockPubkeyConverter()),
 	)
 
 	recovAccnt, err := n.GetAccount(createDummyHexAddress(64))
```

Comment:
Should this be with 32?

Reply:
In this unit test we use a mockPubkeyConverter that internally use hex.Encode and hex.Decode. So the  createDummyHexAddress(64) will actually create a 32 byte long byte array encoded on 64 hex chars. Renamed the parameter inside createDummyHexAddress from chars to hexChars to avoid ambiguity.

Reply:
ok